src/coreclr/vm/debugdebugger.cpp

```diff
@@ -532,182 +535,208 @@ FCIMPL4(void, DebugStackTrace::GetStackFramesInternal,
                 }
             }
 #endif
-            BOOL fPortablePDB = TRUE;
-
             // Check if the user wants the filenumber, linenumber info and that it is possible.
             if (!fIsEnc && fNeedFileInfo)
```

Comment:
Lets add a comment saying that the native diasymreader code has known bugs and limitations when trying to read portable PDBs, so we want to intentionally delegate portable PDBs to the managed parser.